sunkit_pyvista/plotter.py

```diff
@@ -255,8 +271,12 @@ def plot_field_lines(self, field_lines, **kwargs):
         **kwargs :
             Keyword arguments are handed to `pyvista.Plotter.add_mesh`.
         """
+        field_line_meshes = pv.MultiBlock([])
         for field_line in field_lines:
             grid = self._coords_to_xyz(field_line.coords.ravel())
             field_line_mesh = pv.StructuredGrid(grid[:, 0], grid[:, 1], grid[:, 2])
             color = {0: 'black', -1: 'tab:blue', 1: 'tab:red'}.get(field_line.polarity)
             self.plotter.add_mesh(field_line_mesh, color=color, **kwargs)
+            field_line_meshes.append(field_line_mesh)
+
+        self._add_mesh_to_block(block_name='field_lines', mesh=field_line_meshes)
```

Comment:
Codecov says this is missing?

Reply:
I haven't added tests for this yet, I'll add the test in on a separate PR along with other missing tests as mentioned in #33

Reply:
Why not just add them here?

Reply:
I thought adding tests for a separate function unrelated to this PR would be irrelevant. If that's no issue with you then I shall add em here

Reply:
I don't see the issue with adding them here.

Reply:
I've added them here